PyInstaller/hooks/hook-_tkinter.py

```diff
@@ -205,6 +205,34 @@ def _find_tcl_tk(hook_api):
     return tcl_tk
 
 
+def _collect_tcl_modules(tcl_root):
+    """
+    Get a list of TOC-style 3-tuples describing Tcl modules. The modules
+    directory is separate from the library/data one, and is located
+    at $tcl_root/../tclX, where X is the major Tcl version.
+
+    Returns
+    -------
+    Tree
+        Such list, if the modules directory exists.
+    """
+
+    # Obtain Tcl major version.
+    tcl_version = exec_statement(
+        'from %s import Tcl; print(Tcl().eval("info tclversion"))'
+        % modname_tkinter)
```

Comment:
This isn't needed. `modname_tkinter` will be deprecated soon. (It's not even needed anymore; we only support py3 now).

Reply:
OK, I've replaced both the original and the added use of `modname_tkinter`.